doc/classes/Tree.xml

```diff
@@ -191,7 +191,13 @@
 			<return type="TreeItem" />
 			<param index="0" name="position" type="Vector2" />
 			<description>
-				Returns the tree item at the specified position (relative to the tree origin position).
+				Returns the [TreeItem] at the specified position (relative to the tree origin position).
+			</description>
+		</method>
+		<method name="get_item_at_mouse_position" qualifiers="const">
```

Comment:
I believe adding those helper functions are not needed (for both ItemList and Tree). 
You can simply call `tree->get_object_at_position(tree->get_local_mouse_position())` instead.

Reply:
What do you think about renaming them from `get_item_at_mouse_position` to `get_hovered_item`? 

In my opinion these are nice little extensions, and something I have created often for my own editor plugins. I agree they're out of scope for this PR though, and I can remove them. 

Can anyone else weight in?

Reply:
I think any unrelated additional functionality requires a new proposal/PR. The methods you added don't need to be exposed, and as noted above, don't even need to exist.